lib/api/model/events.dart

```diff
@@ -900,16 +901,16 @@ class TypingEvent extends Event {
     required this.recipientIds,
     required this.streamId,
     required this.topic,
-  });
+  }) : assert(isSortedWithoutDuplicates(recipientIds ?? []));
 
   static Object? _readSenderId(Map<Object?, Object?> json, String key) {
     return (json['sender'] as Map<String, dynamic>)['user_id'];
   }
 
   static List<int>? _recipientIdsFromJson(Object? json) {
     if (json == null) return null;
-    return (json as List<Object?>).map(
-      (item) => (item as Map<String, Object?>)['user_id'] as int).toList();
+    return (json as Iterable<Object?>).map(
+      (item) => (item as Map<String, Object?>)['user_id'] as int).toList()..sort();
```

Comment:
commit-message nit:

    events: Sort recipientIds for TypingEvent.

The prefix we've used for changes here is `api:`, so let's stay consistent with that.

A command like `git log --oneline lib/api/model/events.dart` is very handy for looking to see what precedents are for this kind of question.